geanynumberedbookmarks/src/geanynumberedbookmarks.c

```diff
@@ -1452,19 +1452,8 @@ static gboolean Key_Released_CallBack(GtkWidget *widget, GdkEventKey *ev, gpoint
 
 	if(ev->type!=GDK_KEY_RELEASE)
 		return FALSE;
-
-	/* control and number pressed */
-	if(ev->state==4)
-	{
-		i=((gint)(ev->keyval))-'0';
-		if(i<0 || i>9)
-			return FALSE;
-
-		GotoBookMark(doc, i);
-		return TRUE;
-	}
 	/* control+shift+number */
-	if(ev->state==5 || ev->state==21) {
+	if(ev->state & (GDK_CONTROL_MASK | GDK_SHIFT_MASK)) {
```

Comment:
This tests for control or shift, not control and shift

Reply:
Oh word, that wasn't the way I thought it'd work. Thought it was checking right too when I was checking ctrl+number.